library/foundations/scripts/templates/css-variables.template.ts

```diff
@@ -108,12 +109,33 @@ const convertExpressions = (token: TokenOutput): TokenOutput =>
     ? { ...token, value: `Calc(${token.value})` }
     : token;
 
+const getDefaultLayerCssClasses = (layers: TokenLayer[]) => {
+  return layers.flatMap(({ name, isDefault, parameters }) => {
+    console.warn(name);
+    if (
+      ['core', 'components--components'].includes(name) ||
+      name.startsWith('breakpoint-')
+    ) {
+      return [];
+    }
+    const sectionName = parameters[PARAM_SECTION_NAME] ?? name;
+    return isDefault && sectionName ? toKebabCase(sectionName) : [];
+  });
+};
+
 export const renderCSSVariables = (
   { order, layers }: TokenLayers,
   processValue: GeneratorMappingFunction = (_, value) => value
 ) =>
   Buffer.from(`
-@layer ${order.join()};
+@layer ${order.join()},defaultLayers;
+
+@layer defaultLayers {
+  /* Generated to allow the site to have defaults derived by Designer decisions */
+  :root {
+    --defaultLayers: ${getDefaultLayerCssClasses(layers)};
+  }
+}
```

Comment:
New layer to pass the details to the frontend - again you would likely not want that in a prod app, but it allows an end-to-end demo.

Reply:
I think this is good for now. I was thinking of a slightly simpler mechanism: generate a `generated/DEFAULT_BODY_CLASSES` file using the importer (with all the `section-name`  tokens of selected layers) and load it on build time by importing it as a string in root.tsx.
This would make the current mechanism for selecting images to resolve a different `—brand` token, and look for them in a different directory if the brand is different.. 
The event is a little trickier, as we don’t have the ‘default’ event.. but we can think of something (we have the —halloween-event variable.. we just need to know when not to use it)..

Reply:
fixed now